addons/overheating/stringtable.xml

```diff
@@ -914,6 +918,7 @@
             <Korean>제거 불가</Korean>
             <Russian>Несъемный</Russian>
             <French>Inamovible</French>
+            <German>Nicht entfernbar</German>
```

Comment:
I'm not sure if it's correct in this context, so take it with a grain of salt:
```suggestion
            <German>Unentfernbar</German>
```

Reply:
Nicht herausnehmbar [The literal translation would be you can't take the barrel out of the Gun]? In this context, you wouldn't do "un" the 1 on 1 translation would be unremovable [translation wise, it fits, but you just don't use it in this context].